src/templates/base/2019/base.html

```diff
@@ -177,9 +177,7 @@
   </header>
   {% endblock %}
 
-  <div class="container">
-    {% block main %}{% endblock %}
-  </div>
+  {% block main %}{% endblock %}
```

Comment:
I'll be honest, I'm not loving the fact that we now need to specify the container on every page, just because the home page is slightly different - it's like the exception setting the rule, rather than the other way around.

Do we need to do this? How come the current home page is able to break out of this?

Alternatively, if we really do need to do this, then can't we wrap this in a Jinja2 block so it can be overridden by the two index.html pages, but left alone like it was on every other page?:

```suggestion
  {% block container %}
  <div class="container">
    {% block main %}{% endblock %}
  </div>
  {% endblock %}
```

Reply:
The home is different, it uses a lot of CSS grid features and it does not play well with the `.container` helper. 
Therefore, we do need this rule exception.

I'm not a big fan of these `.html` changes too, mostly because the git history shows me as the author of so many (indented) lines and that's not true actually.

Reply:
Well if you go with this new `{% block container %}` suggestion your just need three HTML changes:
1. To `base/2019/base.html` to add this
2. To `base/2019/index.html` to override this to remove the `container` div for this template only.
3. To `base/2020/index.html` to override this to remove the `container` div for this template only.

The rest of your html changes to the other files can be reverted as those other files still will have the container div by default as set in `base.html`.